packages/prisma/seed-app-store.ts

```diff
@@ -240,8 +240,6 @@ export default async function main() {
   }
   // No need to check if environment variable is present, the API Key is set up by the user, not the system
   await createApp("closecom", "closecomothercalendar", ["other"], "closecom_other_calendar");
-  // No need to check if environment variable is present, the API Key is set up by the user, not the system
-  await createApp("sendgrid", "sendgridothercalendar", ["other"], "sendgrid_other_calendar");
```

Comment:
Not sure why this was removed? @hariombalhara

Reply:
seed-app-store-config.json has the same value which is automatically added. So, a manual addition is not required.